src/js/extensions/table/langs.js

```diff
@@ -47,4 +47,11 @@ if (i18n) {
     'Cannot change part of merged cell': '无法更改合并单元格的一部分。',
     'Cannot paste row merged cells into the table header': '无法将行合并单元格粘贴到标题中。'
   });
+
+  i18n.setLanguage(['de', 'de_DE'], {
+    'Merge cells': 'Zellen zusammenführen',
+    'Unmerge cells': 'Zusammenführen rückgängig machen',
```

Comment:
![image](https://user-images.githubusercontent.com/1215767/34924106-bd680eea-f9e4-11e7-9fca-3ec9d018a39a.png)

`Zellen zusammenführen`
`Zusammenführen rückgängig machen`
The texts is too long and has been cut off. Can you think of shorter version of it?
or Is it ok to understand as it is?

Reply:
Why not: Zellen verbinden & Zellen-Verbindung aufheben?

Reply:
Später heißt es ja auch verbundene Zelle

Reply:
I think Zellen-Verbindung aufheben is still too long.
I'll consider widen the context menu if there's no more option.

Reply:
Perhaps: Verbindung aufheben.